arches_for_science/trigger_functions.py

```diff
@@ -0,0 +1,184 @@
+CALCULATE_MULTICARD_PRIMARY_DESCRIPTOR_SINGLE = r"""
+DECLARE
+    fn_config jsonb;
+    name_template text;
+    map_popup_template text;
+    description_template text;
+    all_templates text[];
+    this_template text;
+    loop_index int;
+    descriptor_key text;
+
+    graph UUID;
+    resourceid UUID;
+    node_alias text;
+    alias_with_separators text;
+    node_for_alias UUID;
+    nodegroup_for_alias UUID;
+    localized_string_node_value jsonb;
+    lang text;
+    inner18n_obj_for_lang jsonb;
+    resolved_node_value_for_lang text;
+    working_string text;
+    localized_result_all jsonb;
+    localized_result_name_only jsonb;
+
+BEGIN
```

Comment:
Here's where we'll get git diffs/blames all in one place for trigger logic, instead of only spread across multiple migration files.